cmd/zetae2e/local/post_migration.go

```diff
@@ -0,0 +1,58 @@
+package local
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/fatih/color"
+
+	"github.com/zeta-chain/zetacore/e2e/config"
+	"github.com/zeta-chain/zetacore/e2e/e2etests"
+	"github.com/zeta-chain/zetacore/e2e/runner"
+)
+
+// postMigrationTestRoutine runs post migration tests
+func postMigrationTestRoutine(
```

Comment:
I think this function can be in `migration.go` as related to it and only use it there.

Reply:
I would prefer to leave it out for now, as I am planning to remove it in the future 
and run this separately 
https://github.com/zeta-chain/node/issues/2444